helpers/date.go

```diff
@@ -0,0 +1,11 @@
+package helpers
+
+import (
+	"time"
+)
+
+const DateTimeLayout = "02/01/2006 15:04"
```

Comment:
```suggestion
const (
    DateTimeLayout = "02/01/2006 15:04"
)
```

Should we make it to this?

Reply:
fixed in[f4afac3](https://github.com/carryall/go-google-scraper-challenge/pull/128/commits/f4afac3ed074cf2152f82c7c12ab9330d4617386)